src/avatar/index.tsx

```diff
@@ -1,37 +1,54 @@
 import React from 'react'
 import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
 
-import style from 'avatar/style'
-import Check from 'icon/checkIcon'
 import prefix from '_utils'
 import { color } from '_utils/branding'
+import Badge from 'badge'
+import CheckIcon from 'icon/checkIcon'
+import style from 'avatar/style'
 
-export interface IdCheck {
-  readonly checked?: boolean
-  readonly title?: string
-}
-
-export interface AvatarInterface extends IdCheck {
+export interface AvatarInterface {
   readonly className?: Classcat.Class
   readonly image?: string
   readonly alt?: string
   readonly medium?: boolean
   readonly large?: boolean
+  readonly isIdChecked?: boolean
```

Comment:
`idChecked` instead?

Reply:
why?

Reply:
It's not that easy to read :)

Reply:
Well it's a sort of convention that using 'is' or 'has' clarifies that we're dealing with a boolean.
If we don't go by this rule, ok, but it does make things easier to read/understand.